### PR TITLE
RDAPI-18215: Add additional info and background for API management filters

### PR DESCRIPTION
**Pull request by alexearnshaw**
- Added additional info from RDAPI-18215
- Added some background context about the Client Registry/KPS
- Moved the file to the Policy Reference section

### CODE REVIEW

**Comment by alexearnshaw** [comment]
@andreamussap I've asked the reporter to review the deploy preview output in the JIRA ticket, but meanwhile it could be peer reviewed also (I don't expect much feedback from reporter as the change was quite small)

File: content/en/docs/apigw_polref/api_mgr_ps_filters.md [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
+filter to get information from the Client Registry about a particular organization's, or a particular application's, access to an API.
 
 This filter stores the information in a message attribute (for example, `apimgmt.apiaccess`). You can use this filter within an alert handling policy (or any other policy) to read an organization's or an application's API access easily.
 
 Configure the following fields on the **General settings** tab:
 
 **API ID selector**:
 
-Enter a selector expression with the name of the message attribute that contains the API ID. The value of the selector is expanded at runtime. The default is `${apimgmt.apiproxy.id}`.
+Enter a selector expression with the name of the message attribute that contains the API ID. ID is the key to the API proxy record in the KPS. The value of the selector is expanded at runtime. The default is `${apimgmt.apiproxy.id}`.
```
Comment (andreamussap) [inline content/en/docs/apigw_polref/api_mgr_ps_filters.md]: Suggestion: I think it's better to explain what the field is, then how to use it --> '<This is the> Key to the API proxy record in the KPS. Enter a selector expression...'

(same note for the others below)

**Comment by andreamussap** [comment]
> @andreamussap I've asked the reporter to review the deploy preview output in the JIRA ticket, but meanwhile it could be peer reviewed also (I don't expect much feedback from reporter as the change was quite small)

@alexearnshaw MR is approved with a small suggestion. /Andrea

File: content/en/docs/apigw_polref/api_mgr_ps_filters.md (hunk 1 above)
Comment (alexearnshaw) [reply content/en/docs/apigw_polref/api_mgr_ps_filters.md]: I didn't change this as that sentence does not explain what the API ID selector field is, only what the ID is.

**Pull request merged**